lib/main-app.js

```diff
@@ -3,6 +3,7 @@ const app = electron.app
 const Menu = electron.Menu
 const ipc = electron.ipcMain
 const GhReleases = require('electron-gh-releases')
+const isDev = 'ELECTRON_IS_DEV' in process.env && process.env.NODE_ENV !== 'production'
```

Comment:
I don't think we need to check `'ELECTRON_IS_DEV' in process.env`. Just checking `process.env.NODE_ENV !== 'production'` is definitely enough. 😄

Reply:
Changed as requested